lnwallet/channel.go

```diff
@@ -1448,6 +1500,35 @@ type ForceCloseSummary struct {
 	SelfOutputSignDesc *SignDescriptor
 }
 
+// getCommitTx function take the latest commitment transaction and populate
+// it with witness data.
+func (lc *LightningChannel) getCommitTx() (*wire.MsgTx, error) {
```

Comment:
I think something along the lines of `getSignedCommitTx` would be a more descriptive name for this function.